config/fireBaseConfig.js

```diff
@@ -0,0 +1,12 @@
+// Database configuration
+const config = {
```

Comment:
We should totally avoid committing application settings and secret keys to source control. Is there a reason why we have to do this?